lxd/container.go

```diff
@@ -438,6 +438,19 @@ type container interface {
 	FilePush(type_ string, srcpath string, dstpath string, uid int64, gid int64, mode int, write string) error
 	FileRemove(path string) error
 
+	// Console - Allocate and run a console tty.
+	//
+	// ttynum   - Terminal number to attempt to allocate. Set to -1 to
+	//            allocate the first available tty or to 0 to allocate the
+	//            console.
+	// stdinfd  - File descriptor to read input from.
+	// stdoutfd - File descriptor to write output to.
+	// stderrfd - File descriptor to write error output to.
+	// escape   - The escape character (1 == 'a', 2 == 'b', ...).
```

Comment:
With the comment above, this probably should become:

    Console(terminal *os.File) error